### ISSUE TITLE
Scrambling / Anonymizing calender details

### ISSUE THREAD

**Issue by JacaCroro** [phw198/OutlookGoogleCalendarSync#413]
:memo: Use the below template to report your issue.  
:bulb: Have you upgraded to the latest alpha to check if the problem is already fixed?  
:paperclip: To aid investigation, please provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#posting-your-ogcalsynclog-file)).  

----

**OGCS Version**: v2.6.4  
**Installed or Portable**: Portable

**Problem Description**  

Hi,
Is there any way to anonymize the calender details when upload to goggle? I don't want that the name and email from the participants of a meeting can be found in google calender. I would like to upload only the topic, start/end and date/time

Thanks

**Comment by phw198** [comment]
Don't sync meeting attendees then? `Settings > Sync Options > What >` Uncheck `Attendees`.

To remove anything that's already been synced, you'll need to force a full sync by holding `Shift` and clicking the `Sync` button.

**Comment by scriptPilot** [comment]
@phw198 - The ``SHIFT`` function is very helpful - what about a second button? I just found this Information by chance ...

**Comment by phw198** [comment]
@scriptPilot I did toy with making the full sync more obvious, and I may yet change my mind, but currently I'd prefer to have it a bit more of a "hidden" feature - eg. I don't want people to start doing full syncs as standard to maybe work around a bug that should otherwise be fixed. Plus it's obviously a lot more inefficient...

**Comment by Peggie99** [comment]
So I'm using the Portable OutlookGoogleCalendarSync. I've got calendar entries from Outlook which were synchronized to my google calendar before I checked "Cloak email in Google". 

I've read in several locations (on this forum + in the app) that if i "perform a full sync" using the SHIFT key when clicking the SYNC button, that previously synced calendar items will be updated.
However, this is NOT happening with attendees.... nothing previously created is getting updated in Google.  I've tried portable 2.8.00 and 2.8.1.0: the behviour is the same. 

So I've got Google calendar items - particularly recurring items - going months in the future that I'm afraid to touch, lest Google send emails to all external emails in the list - a "feature" that there is no way to disable, thanks to Google Developers deciding that this is the way it should work.
